jenkins-slaves/jenkins-slave-npm/Dockerfile

```diff
@@ -0,0 +1,33 @@
+#invoke npm in jenkinsfile: sh "scl enable rh-nodejs6 'npm run build'"
+FROM registry.access.redhat.com/openshift3/jenkins-slave-base-rhel7
```

Comment:
Let's remove the server/host from the registry so we can support environments without access to the RH registry (also consistent with other slaves in this repo)